docs/reference/modules/http.asciidoc

```diff
@@ -127,6 +127,44 @@ The maximum number of warning headers in client HTTP responses. Defaults to `unb
 `http.max_warning_header_size`::
 The maximum total size of warning headers in client HTTP responses. Defaults to `unbounded`.
 
+`http.tcp.no_delay`::
+Enable or disable the https://en.wikipedia.org/wiki/Nagle%27s_algorithm[TCP no delay]
+setting. Defaults to `network.tcp.no_delay`.
+
+`http.tcp.keep_alive`::
+Configures the `SO_KEEPALIVE` option for this socket, which
+determines whether it sends TCP keepalive probes.
+Defaults to `network.tcp.keep_alive`.
+
+`http.tcp.keep_idle`:: Configures the `TCP_KEEPIDLE` option for this socket, which
+determines the time in seconds that a connection must be idle before
+starting to send TCP keepalive probes. Defaults to `network.tcp.keep_idle` which
+means to use the system default. May not be greater than 300. Only applicable on
+Linux and macOS, and requires Java 11 or newer.
+
+`http.tcp.keep_interval`:: Configures the `TCP_KEEPINTVL` option for this socket,
+which determines the time in seconds between sending TCP keepalive probes.
+Defaults to `network.tcp.keep_interval` which means to use the system default.
```

Comment:
```suggestion
Defaults to `network.tcp.keep_interval` which means to use the smaller of 300 or the system default.
```

Reply:
Ok, no, I understood #59278 ok and misunderstood this change. This suggestion isn't right.